server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/RangeIT.java

```diff
@@ -184,6 +184,7 @@ public void setupSuiteScopeCluster() throws Exception {
         builders.add(client().prepareIndex("new_index").setSource(Collections.emptyMap()));
 
         indexRandom(true, builders);
+        indexRandomForMultipleSlices("idx", "old_index", "new_index");
```

Comment:
this will index bogus documents for these indices for non-concurrent path as well, which is not required. But since this will happen during the initialization of the cluster, which is once per test classes annotated with `@OpenSearchIntegTestCase.SuiteScopeTestCase`, i think it should be fine.

Reply:
I think it should be fine too and I also don't think there's a way around this for now since the indexing is done in `beforeClass` but the index setting for parameterization is applied during `beforeTests`. See https://github.com/opensearch-project/OpenSearch/pull/11093/files#r1390010637

Reply:
if this needs to be done per test, then we should call `indexRandomForConcurrentSearch` and not `indexRandomForMultipleSlices.

Reply:
Yes I think we are in agreement there. What I'm saying is that index "idx" for example is used in multiple tests, but index only once here in `setupSuiteScopeCluster`. I think it's better to keep it as is (which I think we agree on) for 2 reasons:
1. I don't think it makes sense to index bogus docs into the index for each test method and instead just do it once in the setup
2. Even though we are indexing bogus docs for non-concurrent search too, I think it's more consistent to keep the same segment layout for both concurrent and non-concurrent search for the test